pkg/api/schema/schema.proto

```diff
@@ -204,7 +241,10 @@ service ImmuService {
 	rpc CurrentRoot(google.protobuf.Empty) returns (Root) {
 		option (google.api.http) = {
             get: "/v1/immurestproxy/root"
-        };
+				};
+		option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
```

Comment:
If there no a specific reason to let public CurrentRoot please remove public access

Reply:
@mmeloni The reason `CurrentRoot` is always without auth, is because it seems to be called during `Connected` => since even `Login` is wrapped in a `Connected` call, it would make it impossible to login.